config/schema/field_definitions.json

```diff
@@ -508,5 +508,10 @@
   "content_store_document_type": {
     "description": "The document type as stored in the Content Store",
     "type": "identifier"
+  },
+
+  "part_of_taxonomy": {
+    "description": "Any taxon tagged to the document and any of their ancestor taxons",
```

Comment:
Yes,

So if you have the taxonomy

- Schools
   - Key stage 1
      - Maths
   - Key stage 2
     - Maths

And you tag a document to key stage 1 maths and key stage 2 maths, then:

The field is set to [ks1 maths, ks2 maths, ks1, ks2, schools]

And the document is returned in any of these searches:
 - ks1 maths
 - ks2 maths
 - ks1
 - ks2
 - schools